runtime/compiler/control/J9Options.cpp

```diff
@@ -2205,18 +2212,24 @@ J9::Options::fePostProcessJIT(void * base)
          //
          if (!TR::Options::getCmdLineOptions()->getOption(TR_DisableRampupImprovements) &&
             !TR::Options::getAOTCmdLineOptions()->getOption(TR_DisableRampupImprovements))
-            {
```

Comment:
I'd prefer the braces not be removed, especially since there's an `#ifdef` after the `if`; personally I think it's more futureproof if done:

```
         if (!TR::Options::getCmdLineOptions()->getOption(TR_DisableRampupImprovements) &&
            !TR::Options::getAOTCmdLineOptions()->getOption(TR_DisableRampupImprovements))
            {
#if defined(JITSERVER_SUPPORT)
			compInfo->updateNumUsableCompThreads(_numUsableCompilationThreads);
#else
			_numUsableCompilationThreads = MAX_USABLE_COMP_THREADS;
#endif
			}
```